src/main/java/code/practice/tasks/TransactionProcessing.java

```diff
@@ -0,0 +1,56 @@
+package code.practice.tasks;
+
+import code.practice.model.Transaction;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class TransactionProcessing {
+    private List<Transaction> transactionsList = new ArrayList<>();
+
+    /**
+     * Given list of integers, need to return list which contains all duplicates from original list.
+     * Example:
+     * Given: [3, 5, 1 ,5 ,6 ,5 ,4, 4]
+     * Return: [5, 5, 4]
+     */
+    public List<Integer> getDuplicates(List<Integer> numbers) {
+        List<Integer> listOfDuplicateElements = new ArrayList<>();
+        Set<Integer> uniqueElementsFromNumbersList = new HashSet<>();
+        boolean additionToSetResult;
+        for (int i = 0; i < numbers.size(); i++) {
+            additionToSetResult = uniqueElementsFromNumbersList.add(numbers.get(i));
+            if (!additionToSetResult) {
+                listOfDuplicateElements.add(numbers.get(i));
+            }
+        }
+        return listOfDuplicateElements;
+    }
+
+    //*********************************************************
+    //The next task contains two methods
+    /**
```

Comment:
Please add empty line before method description. CTRL-ALT-L can do this for you.